vuepress/docs/.vuepress/theme/navbar.ts

```diff
@@ -1,6 +1,10 @@
 import type { NavbarConfig } from '@vuepress/theme-default'
 
 export const navbar: NavbarConfig = [
+    {
+        text: 'Logout',
+        link: '/logout',
```

Comment:
Where will this route be defined? Currently https://developer.bcc.no/logout gives a 404, which is where this will link to.